### PR TITLE
EPMRPP-65343: composite attribute go out of the border

### PR DESCRIPTION
**Pull request by oranje322**

### CODE REVIEW

File: app/src/components/filterEntities/entitiesGroup/entitiesGroup.jsx [hunk 1]
```diff
@@ -105,7 +106,7 @@ export class EntitiesGroup extends Component {
             <div key={id} className={cx('entity-item', { vertical })}>
               <EntityComponent
                 entityId={id}
-                smallSize={entitySmallSize}
+                smallSize={entity.id === ENTITY_ATTRIBUTE ? false : entitySmallSize}
```
Comment (chivekrodis) [inline app/src/components/filterEntities/entitiesGroup/entitiesGroup.jsx:109]: It looks not flexible. You bind `smallSize` to false forever. But you can provide this property from upper component which collect all filter property.

**Review by chivekrodis — CHANGES_REQUESTED** [review]

File: app/src/components/filterEntities/entitiesGroup/entitiesGroup.jsx (hunk 1 above)
Comment (oranje322) [reply app/src/components/filterEntities/entitiesGroup/entitiesGroup.jsx:109]: Why am I forever binding `smallSize` to false? In this case, smallSize will only be false for the composite attribute. And that what i want. For others inputs, `smallSize ` inherit from `entitySmallSize  `. And in this case will be true.

Comment (chivekrodis) [reply app/src/components/filterEntities/entitiesGroup/entitiesGroup.jsx:109]: Ok. my bad. I mean bind `false` for `ENTITY_ATTRIBUTE` whenever it used. Maybe it is ok, I just highlight this case.
@AmsterGet pls, check this solution

Comment (AmsterGet) [reply app/src/components/filterEntities/entitiesGroup/entitiesGroup.jsx:109]: `EntitiesGroup` looks like a generic component for entities, and it does not need to know which entities are used in it.
Accordingly, I propose to bind the `false` value for `ENTITY_ATTRIBUTE` inside `EntityInputConditionalAttributes` as we do not support small view for this component.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**